docs(troubleshooting): integrate general_troubleshooting (#8560)
vatesfr/xen-orchestra: docs/docs/troubleshooting.md

If you have issues with XO installed from GitHub (not XOA), [please go to the dedicated section first!](community.md).
:::

## Recommendation

If you think you have a problem with your XOA, start by running the `xoa check` command in your terminal:

```console
$ xoa check
✔ Node version
✔ Disk space for /var
✔ Disk space for /
✔ XOA version
✔ xo-server config syntax
✔ Appliance registration
✔ Internet connectivity
```

> If the result you have is completely different from that, or if error messages are displayed, lost packets, etc., you have, indeed, a problem. 

Reviewer: ```suggestion
If the result you have is completely different from that, or if error messages are displayed, lost packets, etc., you have, indeed, an issue. 
```

Please check that you have enabled virtualization settings in your BIOS or upgrade your hardware.

## User login

> If you're having trouble logging in, please check the logs of `xo-server`.

Reviewer: ```suggestion
If you're having trouble logging in, please check the logs for `xo-server`.
```

Reviewer 2: I kinda disagree with this change :sweat_smile: 

Unless I'm mistaken, the first one means looking at `xo-server`'s logs, the second one means looking through all logs of the system for those related to `xo-server`.

Author: I rephrased it as _"If you're having trouble logging in, look into the `xo-server` logs."_

Reviewer: ok for the rephrasement
